x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_details/input_type_utils.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import {
+  DATASET_FIELD,
+  AGENT_DATASET_FILEBEAT,
+  AGENT_DATASET_METRICBEAT,
+} from '../agent_logs/constants';
+
+export function displayInputType(inputType: string): string {
+  if (inputType === 'logfile') {
```

Comment:
Do we have TS types for any of these input types? Should we? Seems like we have a limited set of acceptable values, or at least a few known ones, and we could add some type safety to them.

Reply:
We do not have, and only logfile and `endpoint` are fix for metrics it's something like `*/metrics`